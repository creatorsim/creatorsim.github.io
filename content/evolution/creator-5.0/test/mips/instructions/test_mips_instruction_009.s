
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
    main:
      li  $t0, 361
      li  $t1, 2
      li  $t2, 5
      li  $t3, 0x44562
      blt $t0, $t2, jump1
      blt $t1, $t0, jump1

    jump2:
      li $t6, 34
      li $v0, 10
      syscall

    jump1:
      li $t4, 11
      li $t5, 555
      blt $t4, $t5, jump2
