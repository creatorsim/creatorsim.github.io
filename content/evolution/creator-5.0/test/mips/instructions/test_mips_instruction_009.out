
cr[PC]:0x34; ir[1,at]:0x1; ir[2,v0]:0xa; ir[8,t0]:0x169; ir[9,t1]:0x2; ir[10,t2]:0x5; ir[11,t3]:0x44562; ir[12,t4]:0xb; ir[13,t5]:0x22b; ir[14,t6]:0x22; keyboard[0x0]:''; display[0x0]:''; 

